pkg/controllers/sidecar/config/config.go

```diff
@@ -98,22 +98,22 @@ func (s *ScyllaConfig) setupScyllaYAML() error {
 	cfg["rpc_address"] = "0.0.0.0"
 	cfg["endpoint_snitch"] = "GossipingPropertyFileSnitch"
 
-	overrideYAMLBytes, err := yaml.Marshal(cfg)
+	overrideBytes, err := yaml.Marshal(cfg)
 	if err != nil {
 		return errors.Wrap(err, "failed to parse override options for scylla.yaml")
 	}
-	scyllaYAMLConfigMapFilteredBytes, err := mergeYAMLs(scyllaYAMLConfigMapBytes, overrideYAMLBytes)
+	overwrittenBytes, err := mergeYAMLs(configFileBytes, overrideBytes)
```

Comment:
The one real change in this patch is changing the first argument of mergeYAMLs in here and the following call, right?

For future changes like this, please split it in two patches in the same pull request
* Add arguments to setupScyllaYAML and use them, no functionality changed.
* Change the the priority by changing the arguments passed to mergeYAML

Reply:
> The one real change in this patch is changing the first argument of mergeYAMLs in here and the following call, right?

Yes, other changes like adding function parameters were needed in order to add tests.

> For future changes like this, please split it in two patches in the same pull request
> 
>     * Add arguments to setupScyllaYAML and use them, no functionality changed.
> 
>     * Change the the priority by changing the arguments passed to mergeYAML

I agree, but in such small changes I tend to forget about such details. When everything is done, splitting them into multiple commits may double the time spent on task.